backends/firrtl/firrtl.cc

```diff
@@ -1076,6 +1106,22 @@ struct FirrtlWorker
 
 		for (auto str : wire_exprs)
 			f << str;
+
+		f << stringf("\n");
+	}
+
+	void run()
+	{
+		// Blackboxes should be emitted as `extmodule`s in firrtl. Only ports are
+		// emitted in such a case.
+		if (module->get_blackbox_attribute())
+		{
+			emit_extmodule();
+		}
+		else
+		{
+			emit_module();
+		}
```

Comment:
OK, already covered by the review of line 402, but doesn't:
```suggestion
		if (module->get_blackbox_attribute())
			emit_extmodule();
		else
			emit_module();
```
just look a lot cleaner?

Reply:
Indeed, done.